scripts/winesapos-convert.sh

```diff
@@ -96,6 +90,13 @@ Include = /etc/pacman.d/chaotic-mirrorlist" | sudo tee -a /etc/pacman.conf
     # shellcheck disable=SC2103
     cd ..
     sudo rm -rf yay
+    # GuestSneezeOSDev: add balenaEtcher
+    git clone https://aur.archlinux.org/balena-etcher.git
```

Comment:
Using the AppImage is easier and more reliable. We also do not use the AUR package for this anymore since we switched to use the AppImage.

Here is how we install it in winesapOS: https://github.com/winesapOS/winesapOS/blob/4.2.0/scripts/winesapos-install.sh#L577